libs/schema-validation/src/main/kotlin/gov/cdc/ocio/reportschemavalidator/utils/CloudSchemaLoaderConfiguration.kt

```diff
@@ -0,0 +1,38 @@
+package gov.cdc.ocio.reportschemavalidator.utils
+
+
+import gov.cdc.ocio.reportschemavalidator.loaders.CloudSchemaLoader
+import io.ktor.server.application.*
+import io.ktor.server.config.*
+
+class CloudSchemaLoaderConfiguration(environment: ApplicationEnvironment){
+    private val schemaLoaderSystem = environment.config.tryGetString("ktor.schema_loader_system")?: ""
+    private val s3Bucket = environment.config.tryGetString("aws.s3.report_schema_bucket") ?: ""
+    private val s3Region = environment.config.tryGetString("aws.s3.report_schema_region") ?: ""
+    private val connectionString = environment.config.tryGetString("azure.blob_storage.connection_string") ?: ""
```

Comment:
Can we please rename this so it's obvious what the connection string is for?  The aws.s3.report_schema_bucket, etc...I like those names.  Can we rename to azure.blob_storage.report_schema_connection_string for example?  Same comment for the container name.

Reply:
Done